src/components/filters/FilterButtonsContainer.tsx

```diff
@@ -0,0 +1,41 @@
+'use client';
+import { Button } from '../ui/button';
```

Comment:
please change to: `from '@/components/ui/button'` for consistency

Reply:
Sorry for that, I have to configure my auto-import on my VS code